go.mod

```diff
@@ -1,18 +1,43 @@
 module github.com/openshift/osd-network-verifier
 
-go 1.16
+go 1.19
 
 require (
-	cloud.google.com/go/compute v1.7.0 // indirect
 	github.com/aws/aws-sdk-go-v2 v1.11.2
 	github.com/aws/aws-sdk-go-v2/config v1.10.3
 	github.com/aws/aws-sdk-go-v2/credentials v1.6.4
 	github.com/aws/aws-sdk-go-v2/service/ec2 v1.24.0
 	github.com/aws/smithy-go v1.9.0
-	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/mock v1.6.0
 	github.com/openshift-online/ocm-sdk-go v0.1.224
 	github.com/spf13/cobra v1.4.0
 	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
 	google.golang.org/api v0.84.0
 )
+
+require (
```

Comment:
hmm, any idea why this is duplicating `require`? It seems like the ones come with the update is not being used by the verifier, such as `grpc ` or `protobuf `

Reply:
Ah it separates out direct/indirect dependencies by default now, I want to say starting Go 1.17? But e.g. https://github.com/openshift/osdctl/blob/master/go.mod

Reply:
found it! It is Go 1.17 https://github.com/golang/go/issues/47733